frontend/src/types/schemaSystem.ts

```diff
@@ -0,0 +1,282 @@
+import { SchemaReviewId } from "./id";
+import { Principal } from "./principal";
+
+export enum RuleLevel {
+  Disabled = "disabled",
```

Comment:
Follow the convention to use UPPERCASE for all ENUM values. In particular, this will be stored in the DB